landing-page.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <meta name="darkreader" content="dark" />
+        <title>namaka</title>
+        <!-- <link rel="stylesheet" href="/demo.css" /> -->
+        <link rel="stylesheet" href="/global.css" />
+
+        <script defer type="importmap-shim" src="/x/importmap.json"></script>
+        <script defer type="module-shim" src="/x/mock.js"></script>
+        <script defer src="/node_modules/es-module-shims/dist/es-module-shims.js"></script>
+    </head>
+    <body>
+        <div>
+            <my-review></my-review>
+           
+            <horizontal-review></horizontal-review>
```

Comment:
indents -- please fix whitespace, use tabs

Reply:
Done... but then, if you can suggest what you use to detect that, I believe it will be of help